test/testcafe/spec-en-leaks/EnglishLeaks_spec.js

```diff
@@ -94,17 +128,21 @@ const setUpResponse = (filePath) => {
   return mock;
 };
 
-async function setup(t, locale) {
+async function setup(t, locale, fileName) {
+  const options = mocksWithInteractionCodeFlow.includes(fileName) ? optionsForInteractionCodeFlow : {};
   const widgetView = new PageObject(t);
   await widgetView.navigateToPage();
```

Comment:
pass `render: false` here to prevent initial render

Reply:
Done